constants_test.py

```diff
@@ -16,8 +16,10 @@
 
 import os
 
-import constants  # pylint: disable=relative-import
-from core.tests import test_utils  # pylint: disable=relative-import
+# pylint: disable=relative-import
```

Comment:
When placed here, this disables the relative-import for the entire file.

Instead, this should be placed only on the line which it affects, like it was before.

/cc @apb7 @kevinlee12 -- might be worth adding this to the list of lint checks; I see a fair amount of overuse of lint pragmas in PRs.

Reply:
Hi, when I add the `relative-import` after each import I get a weird error:
```
ERROR: /home/ich/Programming/oppia/oppia/constants_test.py Imports are incorrectly sorted.
--- /home/ich/Programming/oppia/oppia/constants_test.py:before	2018-07-09 19:17:55.556584
+++ /home/ich/Programming/oppia/oppia/constants_test.py:after	2018-07-09 19:17:58.553864
@@ -16,9 +16,9 @@
 
 import os
 
-import constants # pylint: disable=relative-import
-from core.tests import test_utils # pylint: disable=relative-import
-import feconf # pylint: disable=relative-import
+import constants  # pylint: disable=relative-import
+from core.tests import test_utils  # pylint: disable=relative-import
+import feconf  # pylint: disable=relative-import
 
 
 class ConstantsTests(test_utils.GenericTestBase):
```
The weird thing is that the diffs seems the same.

Reply:
@vojtechjelinek: I am not facing any import error locally for this file. This is the code I am checking:
https://codeshare.io/2peb16
Can you please run this once on your machine? Thanks!

Reply:
Oh, there must be a double space before the comment, I didn't notice that. Still, it is a little bit confusing, because the error says that the order is wrong.

Reply:
Ah, yes, you're right. I'll see what can be done here though in general, we prefer to have two spaces before the comment, that is the way these pragmas work :)

Reply:
Thanks @apb7 -- can we make sure to keep track of this as an item to fix? Lint messages need to be actionable. If they confuse the developer, then they need to be fixed, and this one seems actively misleading.

Reply:
@seanlip: This particular message is not given by us and is provided by isort, a third party library. We cannot override this from our end though I will file an issue for this in upstream. Post that, I will file an issue here to keep a track on the upstream issue.

Reply:
Oh, also, I have noted the above pylint-pragma check in #3905.

Reply:
Thanks!